app/helpers/tasklist_helper.rb

```diff
@@ -36,6 +36,14 @@ def get_sections_count(form, case_log, status = :all)
     subsections.count { |subsection| get_subsection_status(subsection, case_log, form.questions_for_subsection(subsection).keys) == status }
   end
 
+  def get_first_page_or_check_answers(subsection, case_log, form, questions)
+    if is_started?(subsection, case_log, questions)
+      send("case_log_#{subsection}_check_answers_path", case_log)
+    else
+      send("case_log_#{form.first_page_for_subsection(subsection)}_path", case_log)
+    end
+  end
```

Comment:
Not sure if it's nicer, but we could have 1 send statement if we did it like this?

```
path = is_started?(subsection, case_log, questions) ? "case_log_#{subsection}_check_answers_path" : "case_log_#{form.first_page_for_subsection(subsection)}_path"
    send(path, case_log)
 ```